data-service/src/main/java/uk/gov/gchq/palisade/service/data/service/SimpleDataService.java

```diff
@@ -134,10 +143,11 @@ private void auditRequestReceivedException(final ReadRequest request, final Thro
             readerRequest.setOriginalRequestId(request.getOriginalRequestId());
             LOGGER.info("Calling dataReader with: {}", readerRequest);
 
-            final DataReaderResponse readerResult = getDataReader().read(readerRequest,
-                    this.getClass(),
-                    auditRequestReceiver);
+            AtomicLong recordsProcessed = new AtomicLong(0);
+            AtomicLong recordsReturned = new AtomicLong(0);
+            final DataReaderResponse readerResult = getDataReader().read(readerRequest, recordsProcessed, recordsReturned);
             LOGGER.info("Reader returned: {}", readerResult);
+            LOGGER.info("Processed {} and returned {} records", recordsProcessed.get(), recordsReturned.get());
```

Comment:
Combine the two info messages:
```suggestion
            LOGGER.info("Processed {} and returned {} records, reader returned: {}", recordsProcessed.get(), recordsReturned.get(), readerResult);
```